backend/connector-integration/src/utils.rs

```diff
@@ -44,5 +89,2567 @@ pub fn missing_field_err(
     })
 }
 
-pub mod xml_utils;
-pub use xml_utils::preprocess_xml_response_bytes;
+type Error = error_stack::Report<errors::ConnectorError>;
+
+pub trait AccessTokenRequestInfo {
```

Comment:
If there is no polymorphism then don't use traits, instead these can be just be impl based function